torch/onnx/_internal/fx/custom_operator.py

```diff
@@ -0,0 +1,58 @@
+from typing import Callable
+
+import torch
+from .function_dispatcher import _ATENLIB_FUNCTIONS, _OP_OVERLOAD_TO_EXPORTER_KEY_TABLE
+
+ONNX_CUSTOM_OP_LIB = torch.library.Library("onnx_custom", "DEF")
+
+
+def _register_onnx_custom_op_overload(schema: str):
+    """This function decorator registers the wrapped function as a `torch._ops.OpOverload` with the provided `schema`.
+
+    Arguments:
+        schema (str): The schema of the wrapped function in PyTorch's type system; e.g., "my_op(Tensor a, Tensor b) -> Tensor c".
+
+    Example:
+    The following example register the `a_funny_op` function as `torch.ops.onnx_custom.my_op` (type: torch._ops.OpOverload).
+
+        import torch
+        import torch.onnx._internal.fx.custom_operator as fx_custom
+        @fx_custom._register_onnx_custom_op_overload("my_op(Tensor a, Tensor b) -> Tensor c")
+        def a_funny_op(a, b):
+            return a + b
+
+        x = torch.randn(3, 4)
+        y = torch.randn(3, 4)
+        # Now, use op_overload to invoke the custom function.
+        z = torch.ops.onnx_custom.my_op(x, y)
+    """
```

Comment:
Add `Raises` section in docstring?

Reply:
What does `Raises` mean? Do you have an example?

Reply:
Calling out possible exceptions. See https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html for example, and 

https://github.com/pytorch/pytorch/blob/4e396a54e8e9b3f3b9cc11d063fdd4ed25629c0e/torch/_dynamo/eval_frame.py#L588-L592